.gitattributes

```diff
@@ -0,0 +1,8 @@
+/script export-ignore
```

Comment:
@BPScott As far as I can tell the files in this directory aren't needed if this package is included in another project.